### PR TITLE
CI: Use F35

### PR DESCRIPTION
**Pull request by MartinBasti**
Use the latest greatest tools in CI

Signed-off-by: Martin Basti <mbasti@redhat.com>

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

**Review by mkosiarc — APPROVED** [review]

**Review by hjmodi — APPROVED** [review]

**Review by ben-alkov — APPROVED** [review]
LGTM

**Comment by MartinBasti** [comment]
no release notes, CI change only

File: .github/workflows/unittests.yaml [hunk 1]
```diff
@@ -22,11 +22,6 @@ jobs:
             python: "3.8"
             engine: docker
 
-          - name: fedora
-            version: 32
-            python: 3
-            engine: docker
```
Comment (chmeliik) [inline .github/workflows/unittests.yaml:28]: nitpick: commit says remove F33, removes F32

Comment (chmeliik) [reply .github/workflows/unittests.yaml:28]: nvm, merged already

**Pull request merged**